labels.go

```diff
@@ -126,18 +126,17 @@ func Split(s string) []int {
 // The bool end is true when the end of the string has been reached.
 // Also see PrevLabel.
 func NextLabel(s string, offset int) (i int, end bool) {
-	quote := false
 	for i = offset; i < len(s)-1; i++ {
-		switch s[i] {
-		case '\\':
-			quote = !quote
-		default:
-			quote = false
-		case '.':
-			if quote {
-				quote = !quote
+		if s[i] == '.' {
```

Comment:
This should be `if s[i] != '.' { continue }` to reduce indenting.

Reply:
Done.